cmd/boulder-wfe/main.go

```diff
@@ -117,12 +118,11 @@ func main() {
 			// with new RA and SA rpc clients.
 			for {
 				for err := range closeChan {
-					auditlogger.Warning(fmt.Sprintf("AMQP Channel closed, will reconnect in 5 seconds: [%s]", err))
+					auditlogger.Warning(fmt.Sprintf(" [!] AMQP Channel closed, will reconnect in 5 seconds: [%s]", err))
```

Comment:
This was just my OCD trying to standardize the RPC log messages across binaries . (`[c] for client, [s] for server, [!] for warning/error`)